cmd/list/orgs/orgs.go

```diff
@@ -42,7 +42,11 @@ var OrgsCmd = &cobra.Command{
 				orgOut = fmt.Sprintf("['%s'", org)
 			}
 		}
-		orgOut = fmt.Sprintf("%s]", orgOut)
+		if orgOut != "" {
+			orgOut = fmt.Sprintf("%s]", orgOut)
+		} else {
+			orgOut = "[]"
+		}
```

Comment:
Consider modifying the code to make use of the `strings.Join` function instead. Something like
```go
orgOut := fmt.Sprintf("[%s]", strings.Join(orgs, ","))
```
should be able to replace lines 36 to 49

Reply:
The same thing can probably be applied to the repos file.